README.md

```diff
@@ -4,88 +4,7 @@ cmark-scala provides [Scala Native](http://www.scala-native.org/) bindings for [
 The bindings were directly derived from [cmark.h](https://github.com/jgm/cmark/blob/master/src/cmark.h). Comments were retained and adapted if necessary. The naming of functions and their encapsulation follows Scala's conventions. Note that `*_new` functions were renamed to `create` as to prevent name collisions with the eponymous Scala keyword.
 
 ## Example
-```scala
```

Comment:
Perhaps we could provide a trimmed-down code example here to showcase the basic functionality and add a link to the test cases you wrote?